subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -23,43 +24,42 @@
 /**
  * BEWARE! This plugin is in incubating state, so its API may change in the future!
  * The plugin applies following plugins:
- *
+ * <p>
  * <ul>
- *     <li>{@link ShipkitConfigurationPlugin}</li>
- *     <li>{@link GitOriginPlugin}</li>
+ * <li>{@link ShipkitConfigurationPlugin}</li>
+ * <li>{@link GitOriginPlugin}</li>
  * </ul>
- *
+ * <p>
  * and adds following tasks:
- *
+ * <p>
  * <ul>
- *     <li>checkoutVersionUpgradeBaseBranch - checkouts base branch - the branch to which version upgrade should be applied through pull request</li>
- *     <li>pullUpstream - syncs the fork on which we perform version upgrade with the upstream repo</li>
- *     <li>checkoutVersionUpgradeVersionBranch - checkouts version branch - a new branch where version will be upgraded</li>
- *     <li>replaceVersion - replaces version in build file, using dependency pattern</li>
- *     <li>commitVersionUpgrade - commits replaced version</li>
- *     <li>pushVersionUpgrade - pushes the commit to the version branch</li>
- *     <li>createPullRequest - creates a pull request between base and version branches</li>
- *     <li>performVersionUpgrade - task aggregating all of the above</li>
+ * <li>checkoutVersionUpgradeBaseBranch - checkouts base branch - the branch to which version upgrade should be applied through pull request</li>
+ * <li>pullUpstream - syncs the fork on which we perform version upgrade with the upstream repo</li>
+ * <li>checkoutVersionUpgradeVersionBranch - checkouts version branch - a new branch where version will be upgraded</li>
+ * <li>replaceVersion - replaces version in build file, using dependency pattern</li>
+ * <li>commitVersionUpgrade - commits replaced version</li>
+ * <li>pushVersionUpgrade - pushes the commit to the version branch</li>
+ * <li>createPullRequest - creates a pull request between base and version branches</li>
+ * <li>performVersionUpgrade - task aggregating all of the above</li>
  * </ul>
- *
+ * <p>
  * Plugin should be used in client projects that want to have automated version upgrades of some other dependency, that use the producer version of this plugin.
  * Project with the producer plugin applied would then clone a fork of client project and run './gradlew performVersionUpgrade -Pdependency=${group:name:version}' on it.
- *
+ * <p>
  * Example of plugin usage:
- *
+ * <p>
  * Configure your 'shipkit.gradle' file like here:
- *
- *      apply plugin: 'org.shipkit.upgrade-dependency'
- *
- *      upgradeDependency{
- *          baseBranch = 'release/2.x'
- *          buildFile = file('build.gradle')
- *      }
- *
+ * <p>
+ * apply plugin: 'org.shipkit.upgrade-dependency'
+ * <p>
+ * upgradeDependency{
```

Comment:
Probably your IDE removed whitespaces here. It would be useful to keep it, it's more readable this way.